core/src/com/softwarearchitecture/math/Rectangle.java

```diff
@@ -0,0 +1,257 @@
+package com.softwarearchitecture.math;
+
+public class Rectangle {
```

Comment:
Does all the methods and variables need to be public?

Reply:
i get error when private